### ISSUE TITLE
Start time for alltheplaces-serve weekly build too soon as ATP build is taking longer to build each week?

### ISSUE THREAD

**Issue by davidhicks** [jleedev/alltheplaces-serve#7]
Hi @jleedev I've noticed this week that the map at https://www.alltheplaces.xyz/map/#1/0/0 hasn't updated within it's usual slightly delayed timeframe. ATP has now hit 2.9 million POIs with a significant increase in total number of spiders (and thus overall crawl time) in the past 6 months. I'm wondering whether the weekly build of alltheplaces-serve  is scheduled to occur too soon and is thus building from week old ATP builds, as the ATP weekly build is now taking longer to complete?

Or perhaps ATP has broken the weekly build of alltheplaces-serve in some other way?

**Comment by davidhicks** [comment]
Also hopefully we can now more easily see the changes to ATP weekly build durations via proposed change https://github.com/alltheplaces/alltheplaces/pull/5567

**Comment by jleedev** [comment]
It's broken, I suspect that stadt_zuerich_ch has caused the tippecanoe invocation to tip over due to the unreasonable density of points relative to the other files. I've just started to look at this.

But anyway, I had been manually rolling the cron forwards periodically, which is probably good enough outside of anything more sophisticated.

**Comment by davidhicks** [comment]
Thanks for the update! I noticed that stadt_zuerich_ch hasn't changed that much in size recently though. On the [7th of January](https://data.alltheplaces.xyz/runs/2023-01-07-13-32-28/stats/_results.json) it returned 44972 points. On the [15th of May](https://data.alltheplaces.xyz/runs/2023-05-15-03-47-28/stats/_results.json) it returned 45427 points. The [most recent build](https://data.alltheplaces.xyz/runs/2023-07-15-13-31-57/stats/_results.json) that appears to have now failed has 45535 points. It's possible the limit has finally been reached somewhere in the last 100 or so points.

I wonder however whether the western_union spider which I recently fixed may have broken something, as this is the most significant change I can think of occurring in the last week. It's now an enormous 300MB+ JSON file with >500k points and also has very high density in some cities, albeit probably not as much as stadt_zuerich_ch which captures... individual lamp poles. The previous version of the western_union spider was only returning something like 427428 points on the [15th of May](https://data.alltheplaces.xyz/runs/2023-05-15-03-47-28/stats/_results.json) so it has always been big, but perhaps it's just a little too big now with more fields of data being captured than ever before.
